### DISCUSSION TITLE
out of bounds for axis 0

### DISCUSSION THREAD

**Discussion by GeoHager16** [pycaret/pycaret#4007] [Show and tell]
Hi iam trying to run this code 
# df_pivoted = df.pivot(columns='x', index='y', values='a_2')

df_set = df[['x','y','a_2']]
df_set = df_set.apply(pd.to_numeric)
df_pivoted = df_set.pivot(index='x', columns='y', values='a_2')

ax = sns.heatmap(data=df_pivoted, cmap='Greys', cbar=True, cbar_kws={'label': 'z'})
ax.invert_yaxis()

plt.savefig('out.jpg', dpi=240, facecolor='w', edgecolor='w',
        orientation='portrait', transparent=False, pad_inches = 0.2, bbox_inches='tight') 
plt.show()
but iam having this error: 
IndexError: index 694373722 is out of bounds for axis 0 with size 694340772

**Comment by celestinoxp** [comment]
this is related to indice size... try insert this line to redefine the index:
df.reset_index(drop=True, inplace=True)

**Reply by GeoHager16** [reply]
still getting the same error:

index 694369844 is out of bounds for axis 0 with size 694340772
<Figure size 432x288 with 0 Axes>

**Reply by celestinoxp** [reply]
can you give your code with the reset index?
